espnet/asr/pytorch_backend/asr.py

```diff
@@ -453,6 +453,9 @@ def train(args):
 
     # Evaluate the model with the test dataset for each epoch
     trainer.extend(CustomEvaluator(model, valid_iter, reporter, device, args.ngpu))
+    if args.save_interval_iters > 0:
```

Comment:
I think it is safe to use `if else` instead of set the CustomEvaluator twice.

Reply:
OK